config/hp.yml

```diff
@@ -13,6 +13,12 @@ products:
 term_browser: custom
 
 entries:
+- prefix: /releases/hpoa/
+  replacement: https://github.com/obophenotype/human-phenotype-ontology/releases/download/v
+  tests:
+  - from: /releases/hpoa/2022-02-14/genes_to_phenotype.txt
+    to: https://github.com/obophenotype/human-phenotype-ontology/releases/download/v2022-02-14/genes_to_phenotype.txt
```

Comment:
what does this do?

Reply:
the test is just for OBO PURL system to have something to check whether the redirect works the way intended. Not affecting anything around Purls, users etc..